org.eclipse.m2e.core.ui/plugin.xml

```diff
@@ -674,4 +686,31 @@
    <extension point="org.eclipse.m2e.core.archetypeCatalogs">
       <remote  url="https://repo1.maven.org/maven2/archetype-catalog.xml" description="Maven Central"/>
    </extension>
+   <extension
+         point="org.eclipse.ui.menus">
+      <menuContribution
+            allPopups="false"
+            locationURI="menu:project">
+            <command
+                  commandId="org.eclipse.m2e.core.ui.command.updateProject"
+                  style="push">
+               <visibleWhen
+                     checkEnabled="false">
+                   <or>
+                       <with variable="selection">
```

Comment:
This does not seems to work with the Project Explorer at least. Selecting a project in Project Explorer without editor open doesn't show the menu.